runtime/CMakeLists.txt

```diff
@@ -3,13 +3,17 @@ set(CHAINER_COMPILER_COMMON_DIR ${CHAINER_COMPILER_ROOT_DIR}/common)
 set(GOOGLETEST_INCLUDE_DIRS ${CHAINER_COMPILER_ROOT_DIR}/googletest/googletest/include)
 set(GSLLITE_INCLUDE_DIRS ${CHAINER_COMPILER_ROOT_DIR}/gsl-lite/include)
 
+if(MSVC)
+    find_package(PythonInterp 3)
```

Comment:
Thanks for pointing this out! I think we should move this to the top-level CMakeLists.txt without `if(MSVC)` guard and replace all `python3`.